src/conversions.jl

```diff
@@ -11,6 +11,14 @@ Returns an equivalent `DateTime` without any `TimeZone` information.
 """
 DateTime(zdt::ZonedDateTime) = localtime(zdt)
 
+"""
+    Date(::ZonedDateTime) -> Date
+
+Returns an equivalent `Date` without any `TimeZone`, or time information.
+"""
+Dates.Date(zdt::ZonedDateTime) = Date(DateTime(zdt))
```

Comment:
I've been intending on dropping `DateTime(::ZonedDateTime)` in favor of `localtime(::Type{DateTime}, ::ZonedDateTime)` or something similar. This is definitely a lossy operation and it's important to communicate the implicit time zone.

Reply:
https://github.com/JuliaTime/TimeZones.jl/issues/125

Reply:
I would really dislike that.
Often I just have a collection of `DateTime`,
one per day.
and I want to plot them (cf #184),
or I want to display them in a table,
and I just want to get rid of extra information.
I don't actually care about the time or even the exact date often.

I think it pretty clear to the user that  it is losing information.

Reply:
Looking at that issue again `Date(zdt, UTC)` / `Date(zdt, Local)` isn't bad

Reply:
I am happy to see this change when we resolve #125 for `DateTime`s.
Til then this match the API we have

Reply:
Let's just make the change now? Both the `UTC` and `Local` types already exist

Reply:
If you don't want to take on the change yourself I can make the remainder of the changes. I don't think I would merge this as is.

Reply:
I do want to merge as is, because the status quo of matching how `DateTime` works is known.
All other things are not known and we might want to do a little design and thinking.

### OTOH:
 - `Date(zdt, Local)` and `Date(zdt, UTC)` // `DateTime(zdt, Local)` and `DateTime(zdt, UTC)` do sound right to me though.
 - A bunch of places that do `DateTime(ztd)` will need to be changed due to deprecation. and any places that do `Date(DateTime(ztd))` will be a subset of those. So doing both at once would mean  we can go staight to `Date(ztd, Local)`. 

I think I am convinced.

Reply:
> I think I am convinced

Convinced enough to do the work? ;). As I said I can do the leg work if you prefer.

Reply:
I'm not sure how deep I would have to go,
so if you know exactly what you have to change be my guest.

Otherwise this might sit til after neurips

Reply:
Waiting til after NeurIPS might be fine though.
We've lived this long